cypress/integration/06.Search.feature

```diff
@@ -8,7 +8,7 @@ Feature: Search
     And we have the following posts in our database:
       | Author        | id | title                                         | content                                 |
       | Brianna Wiest | p1 | 101 Essays that will change the way you think | 101 Essays, of course!                  |
-      | Brianna Wiest | p1 | No searched for content                       | will be found in this post, I guarantee |
+      | Brianna Wiest | p2 | No searched for content                       | will be found in this post, I guarantee |
```

Comment:
thanks for fixing this, I must have been crazy to create two posts with the same id!!